src/utils/themes.ts

```diff
@@ -58,7 +63,10 @@ export const lightTheme: Theme = {
     fill: '#D8E6EA',
     color: '#2A6475',
     activeFill: '#1DE9AC',
-    activeColor: '#1DE9AC'
+    activeColor: '#1DE9AC',
+    label: {
+      outlineColor: new Color(255, 255, 255)
```

Comment:
Let's not use this color constructor. I would prefer primatives on these for simplicity.